composer/loggers/mosaicml_logger.py

```diff
@@ -88,6 +89,8 @@ def log_metrics(self, metrics: Dict[str, Any], step: Optional[int] = None) -> No
         self._log_metadata(metrics)
 
     def after_load(self, state: State, logger: Logger) -> None:
+        # Log composer version
+        self._log_metadata({'composer_version': composer_version})
```

Comment:
I don't think this should be here. It should probably just be directly in `composer.__init__`?

Reply:
Thanks for the suggestion! Fixed 👍